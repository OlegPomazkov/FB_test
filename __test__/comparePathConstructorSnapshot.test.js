var Enzyme = require('enzyme');
var Adapter = require('enzyme-adapter-react-16');

Enzyme.configure({ adapter: new Adapter() });

var React = require('react');
var renderer = require('react-test-renderer');
var configureStore = require('redux-mock-store').default;

var PathConstructor = require('../app/containers/PathConstructor.js');

describe('Check application Snapshot to be the same ==========> ',()=>{
  const initialState = {
    mapReducer: {
      points: [],
      pathMap: {},
      placemarks: [],
      lines: [],
      isMap: false
    }
  };
  const mockStore = configureStore();
  let store = mockStore(initialState); 

  it('Capturing Snapshot of PathConstructor  ---------------> ', () => {
    const renderedValue =  renderer.create(<PathConstructor store={store}/>).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
