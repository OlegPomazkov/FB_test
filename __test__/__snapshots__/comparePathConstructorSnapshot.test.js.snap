// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check application Snapshot to be the same ==========>  Capturing Snapshot of PathConstructor  --------------->  1`] = `
<div
  className="application"
>
  <div
    className="side__panel"
  >
    <div
      className="input__container"
    >
      <p
        className="input__header"
      >
        Новая точка маршрута
      </p>
      <input
        className="input__input"
        onKeyDown={[Function]}
        placeholder="Введите наименование"
        type="text"
      />
    </div>
    <div
      className="list__container"
    >
      <p
        className="list__header"
      >
        Текущий маршрут
      </p>
      <p
        className="list__placeholder"
      >
        У вас пока нет точек на маршруте
      </p>
    </div>
  </div>
  <div
    className="map__container"
    onClick={undefined}
  >
    <p
      className="map__placeholder"
    >
      Карта загружается ... 
    </p>
    <div
      className="map__map"
      id="pathMapId"
      style={
        Object {
          "height": 377,
          "width": 400,
        }
      }
    />
  </div>
</div>
`;
